### PR TITLE
Edits to README.md

### PR DESCRIPTION
**Pull request by awu556**
Hey guys, I just corrected some grammar stuff while reading through the README.md file, reworded some sentences, and updated some of the Discord names. Let me know what you think about the changes.

### CODE REVIEW

**Comment by SomeMoosery** [comment]
LGTM, I'll merge once the tests pass. Thanks for taking the time to update this - it was long overdo!

File: client/package.json [hunk 1]
```diff
@@ -6,19 +6,17 @@
   "dependencies": {
     "@blockstack/connect": "^2.3.1",
     "@fortawesome/free-brands-svg-icons": "^5.12.1",
-    "@material-ui/core": "^4.9.7",
```
Comment (SomeMoosery) [inline client/package.json]: For some reason, looks like this got removed / the `package.json` got switched up which is why build is failing

Comment (awu556) [reply client/package.json]: Oops, my bad. I installed @material-ui core at another branch, I thought it wasn't being used anywhere so I removed it. I'll reinstall it as a dependency and then re-push another branch.

Comment (SomeMoosery) [reply client/package.json]: No prob! Looks like now this just created a merge conflict on your PR - you'll just have to rebase off the current master to fix that.

[Here's a guide to point you in the right direction](https://stackoverflow.com/questions/15602037/git-rebase-upstream-master-vs-git-pull-rebase-upstream-master) - feel free to lmk if you have any questions!

**Pull request merged**